README.md

```diff
@@ -820,6 +820,10 @@ Options:
   --ftx-api-key TEXT              Your FTX API key
   --ftx-api-secret TEXT           Your FTX API secret
   --ftx-account-tier TEXT         Your FTX Account Tier
+  --ftxus-organization TEXT       The name or id of the organization with the FTX.US module subscription
+  --ftxus-api-key TEXT            Your FTX.US API key
```

Comment:
> FTX.US

nit but I think we should be consistent with the naming, suggest make all `FTXUS` following the Lean market